lib/MusicBrainz/Server/Report/ReleasesWithDownloadRelationships.pm

```diff
@@ -12,11 +12,19 @@ sub query {
         FROM
             release r
             JOIN artist_credit ac ON r.artist_credit = ac.id
-            JOIN l_release_url lru ON lru.entity0 = r.id
-            JOIN link ON lru.link = link.id
-            JOIN medium ON r.id = medium.release
-        WHERE link.link_type IN (74, 75)
-          AND medium.format != 12
+        WHERE EXISTS (
+              SELECT TRUE
+              FROM medium
+              WHERE medium.release = r.id
+              AND medium.format != 12
+          )
+          AND EXISTS (
+              SELECT TRUE
+              FROM l_release_url lru
+                JOIN link ON lru.link = link.id
+              WHERE lru.entity0 = r.id 
+                AND link.link_type IN (74, 75)
```

Comment:
The indentation here is wonky

Reply:
@reosarevok: Made a suggestion with commit https://github.com/metabrainz/musicbrainz-server/pull/1140/commits/f3e00691883bbd9acf4f263f1bbd81c2b67899a7 which sticks back to initial indent style.

Reply:
@mwiencek: are you okay with the style @yvanzo chose?

Reply:
Sure. I prefer right-aligning the leftmost keyword, but it's a personal preference. I can read the query fine as it is.